addons/mail/wizard/mail_template_preview.py

```diff
@@ -78,7 +78,7 @@ def _compute_mail_template_fields(self):
                 copy_depends_values['resource_ref'] = '%s,%s' % (self.resource_ref._name, self.resource_ref.id)
                 mail_values = mail_template.with_context(template_preview_lang=self.lang).generate_email(
                     self.resource_ref.id,
-                    self._MAIL_TEMPLATE_FIELDS
+                    self._MAIL_TEMPLATE_FIELDS + ['partner_to']
```

Comment:
Should come with a unit test :)

Reply:
Indeed, I added recipient tests in test_mail_template_preview.py: TestMailTemplateTools where there was already a test about the template preview. Also, reading again my notes, it seems we could backport it to at least 15. So I will test the different version and make a fix for an earlier version.